luigi_pipeline/configs/seqr-mito-loading-local.cfg

```diff
@@ -4,19 +4,33 @@
 sample_type = WGS
 genome_version = 38
 dont_validate=True
-source_paths = ["test_ten_cmg_mito_combined.mt"]
+source_paths = ["../test_ten_cmg_mito_combined.mt"]
+; source_paths = ["gs://seqr-datasets/cmg_mito/test_ten_cmg_mito_combined.mt"]
```

Comment:
There is a ton of commented out code in this file. We should either use those values instead, add helpful comments explaining why we left them in, or take them out